pkg/cue/README.md

```diff
@@ -1,10 +0,0 @@
-# Develop Tips
-
-The following packages need to be tested without external/tool dependencies, So that the cue-sh/unity can do verification
```

Comment:
we do need to make such kinds of packaged be tested included by CUE  community.

Reply:
This is removed to workflow repo.